pkgs/development/python-modules/greenback/default.nix

```
@@ -0,0 +1,46 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  asgiref,
+  greenlet,
+  importlib-metadata,
+  setuptools,
+  sniffio,
+  outcome,
+}:
+
+buildPythonPackage rec {
+  pname = "greenback";
+  version = "1.2.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "oremanj";
+    repo = "greenback";
+    rev = "v${version}";
```

These are tags upstream as well, so:


```suggestion
    tag = "v${version}";
```

Avoids problems when upstream (accidentally) pushes a branch with the same name.